### PR TITLE
Add phoenix 1985205

### PR DESCRIPTION
**Pull request by rnestler**
The package name is manufacturer_part-number as suggested by IPC. The device uses the name from the datasheet: https://media.digikey.com/pdf/Data%20Sheets/Phoenix%20Contact%20PDFs/COMBICON%20Spring-Cage%20PCB%20Term.%20Blocks.pdf

PHOENIX_1985205

### CODE REVIEW

File: dev/022a90f1-9aed-451a-8b94-32ff0ceb4b1d/device.lp [hunk 1]
```diff
@@ -0,0 +1,15 @@
+(librepcb_device 022a90f1-9aed-451a-8b94-32ff0ceb4b1d
+ (name "Phoenix PTSA 1.5/3-3.5-Z")
+ (description "PCB termination blocks, offset pins, 3.5 mm pitch")
```
Comment (dbrgn) [inline dev/022a90f1-9aed-451a-8b94-32ff0ceb4b1d/device.lp]: Use the same description as the package:

```suggestion
 (description "Spring-Cage PCB Termination Block with 3 positions, 45 degree mount with 3.5mm pitch")
```

File: pkg/623398d0-43f3-4601-9cc4-cbce192a8521/package.lp [hunk 2]
```diff
@@ -0,0 +1,81 @@
+(librepcb_package 623398d0-43f3-4601-9cc4-cbce192a8521
+ (name "PHOENIX_1985205")
+ (description "Spring-Cage PCB Termination Block with 3 positions, 45 degree mount with 3.5mm pitch")
+ (keywords "spring-cage")
```
Comment (dbrgn) [inline pkg/623398d0-43f3-4601-9cc4-cbce192a8521/package.lp]:
```suggestion
 (keywords "combicon,spring-cage")
```

File: pkg/623398d0-43f3-4601-9cc4-cbce192a8521/package.lp [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+ (author "Raphael Nestler")
+ (version "0.1")
+ (created 2019-06-14T23:41:25Z)
+ (deprecated false)
+ (category 6183d171-e810-475a-a568-2a270aff8f5e)
```
Comment (dbrgn) [inline pkg/623398d0-43f3-4601-9cc4-cbce192a8521/package.lp:9]: If we want to wait for https://github.com/LibrePCB-Libraries/LibrePCB_Base.lplib/pull/30/files:

```suggestion
 (category b724b55e-0f80-4cf3-aa9f-e2a87e02bd19)
```

(Alternatively we can put it in the pin socket category for now and change the category once the other PR is merged.)

File: pkg/623398d0-43f3-4601-9cc4-cbce192a8521/package.lp [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+   (width 0.2) (fill false) (grab_area true)
+   (vertex (position -5.0 1.0) (angle 0.0))
+   (vertex (position -5.0 2.0) (angle -260.0))
+   (vertex (position -3.5 2.0) (angle 0.0))
+   (vertex (position -3.5 1.0) (angle -180.0))
+   (vertex (position -5.0 1.0) (angle 0.0))
+  )
+  (stroke_text 1cc48d05-e905-4ec4-8dc1-3198bdb23cfe (layer top_names)
+   (height 1.0) (stroke_width 0.2) (letter_spacing auto) (line_spacing auto)
+   (align center bottom) (position 0.0 6.5) (rotation 0.0)
```
Comment (dbrgn) [inline pkg/623398d0-43f3-4601-9cc4-cbce192a8521/package.lp]:
```suggestion
   (align center bottom) (position 0.0 7.0) (rotation 0.0)
```

File: pkg/623398d0-43f3-4601-9cc4-cbce192a8521/package.lp [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+   (auto_rotate true) (mirror false) (value "{{NAME}}")
+  )
+  (stroke_text 3b82662b-a60b-48a2-a3be-dcf95ee5fd7e (layer top_values)
+   (height 1.0) (stroke_width 0.2) (letter_spacing auto) (line_spacing auto)
+   (align center top) (position 0.0 -6.5) (rotation 0.0)
```
Comment (dbrgn) [inline pkg/623398d0-43f3-4601-9cc4-cbce192a8521/package.lp]:
```suggestion
   (align center top) (position 0.0 -7.0) (rotation 0.0)
```

**Review by dbrgn — CHANGES_REQUESTED** [review]
:+1:

File: pkg/623398d0-43f3-4601-9cc4-cbce192a8521/package.lp (hunk 3 above)
Comment (rnestler) [reply pkg/623398d0-43f3-4601-9cc4-cbce192a8521/package.lp:9]: I think we leave it for now and may change it later, if https://github.com/LibrePCB-Libraries/LibrePCB_Base.lplib/pull/30 gets merged.

**Pull request merged**